code/game/objects/random/loot.dm

```diff
@@ -335,30 +335,40 @@
 		/obj/item/storage/pill_bottle/mortaphenyl = 3,
 		/obj/item/storage/pill_bottle/happy = 2,
 		/obj/item/storage/pill_bottle/zoom = 2,
+		/obj/item/reagent_containers/pill/cocaine = 3,
+		/obj/item/reagent_containers/pill/heroin = 2,
+		/obj/item/reagent_containers/hypospray/autoinjector/stimpack = 2,
+		/obj/item/reagent_containers/hypospray/autoinjector/berserk = 1,
+		/obj/item/reagent_containers/hypospray/autoinjector/impedrezene = 0.4,
+		/obj/item/reagent_containers/hypospray/autoinjector/night_juice = 0.3,
+		/obj/item/storage/box/syndie_kit/syringe_gun = 0.5,
+		/obj/item/reagent_containers/syringe/drugs = 1,
+		/obj/item/reagent_containers/inhaler/space_drugs = 2,
+		/obj/item/grenade/spawnergrenade/manhacks = 0.2,
+		/obj/item/storage/box/smokes = 0.7,
+		/obj/item/grenade/napalm = 0.4,
 		/obj/item/reagent_containers/glass/beaker/vial/random/toxin = 1,
-		/obj/item/contraband/poster = 5,
 		/obj/item/material/knife/butterfly = 2,
 		/obj/item/material/butterflyblade = 3,
 		/obj/item/material/butterflyhandle = 3,
 		/obj/item/material/wirerod = 3,
 		/obj/item/melee/baton/cattleprod = 1,
 		/obj/item/material/knife/tacknife = 1,
+		/obj/item/material/knife/bayonet = 2,
+		/obj/item/device/multitool/hacktool = 0.3,
 		/obj/item/material/kitchen/utensil/knife/boot = 2,
 		/obj/item/storage/secure/briefcase/money = 1,
+		/obj/item/gun/projectile/pistol = 0.2,
+		/obj/item/gun/projectile/revolver/derringer = 0.4,
+		/obj/item/gun/projectile/shotgun/improvised/sawn = 0.3,
+		/obj/item/storage/pill_bottle/spotlight = 1,
+		/obj/item/pen/reagent/paralysis = 1,
+		/obj/item/pen/reagent/poison = 1,
```

Comment:
I don't think these two should be there, especially not the parapen
don't want crew parapening each other on accident without any feedback as to what happened
since like it just appears to be a pen to non-antags, and the injecting action does not have an equivalent for normal pens

Reply:
Agree for the distinguishability issue

Reply:
I will remove them both, no worries at all.

Reply:
still need to be removed

Reply:
Removed, sorry about that. I thought I had done it.